python/huggingfaceserver/huggingfaceserver/vllm/vllm_completions.py

```diff
@@ -394,6 +392,73 @@ def _validate_prompt_and_tokenize(
         else:
             return input_ids, input_text
 
+    def _tokenize_prompt_input_or_inputs(
```

Comment:
noob question - Did we copy it because we cannot import this private function? https://github.com/vllm-project/vllm/blob/main/vllm/entrypoints/openai/serving_engine.py#L320
Does it provide any benefits compared to the old code?

Reply:
Yes. It's more of a refactoring for separation of concern.